### PR TITLE
Added image publishing flows

### PR DESCRIPTION
**Pull request by MMoisa**
https://github.com/Financial-Times/methode-image-model-mapper/pull/4
http://git.svc.ft.com/projects/CP/repos/ingester/pull-requests/75/overview

### CODE REVIEW

File: kube-config-files/content-ingester-service.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          value: "9092"
+        - name: KAFKA_GROUPNAME
+          value: "content"
+        - name: KAFKA_SYSTEM_ID
+          value: "ContentIngester"
+        - name: KAFKA_TOPIC_NAME
+          value: "CmsPublicationEvents"
+        - name: WHITELIST
+          value: "^http://(methode|wordpress|brightcove)(-article|-image-model|-list|-image|-video-model|-content-placeholder)?(-internal-components)?(-transformer|-mapper)(-pr|-iw)?(-uk-.*)?\\.svc\\.ft\\.com(:\\d{2,5})?/(content|image/model|image-set/model|lists|video/model|internalcomponents)/[\\w-]+.*$" 
+        - name: WRITER_HEADER
```
Comment (tamas-molnar) [inline kube-config-files/content-ingester-service.yaml]: Is this still needed?

File: kube-config-files/content-ingester-service.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          value: "PostPublicationEvents"
+        - name: KAFKA_PROXY_URL
+          value: "kafka:8082"
+        - name: WRITER_URL
+          value: "document-store-api:8080"
+        - name: ZOOKEEPER_HOST
+          value: "kafka"
+        - name: ZOOKEEPER_PORT
+          value: "2181"
+        - name: UPP_GATEWAY_PATH
```
Comment (tamas-molnar) [inline kube-config-files/content-ingester-service.yaml]: Can we get rid of this?

File: kube-config-files/methode-image-model-mapper-service.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+          value: "-Xms256m -Xmx256m -XX:+UseG1GC -server"
+        - name: KAFKA_PROXY
+          value: "kafka:8082"
+        - name: S3_BUCKET
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: aws.s3.bucket
+        - name: S3_URL
+          value: "http://$S3_BUCKET.s3.amazonaws.com/"
```
Comment (sbuliarca) [inline kube-config-files/methode-image-model-mapper-service.yaml]: Is this a global S3 bucket used in other places, or is it dedicated especially for this app?

File: global-config/global-config.yaml [hunk 4]
```diff
@@ -22,4 +22,5 @@ metadata:
 data:
   mongodb.address: mongodb:27017
   api.host: test.api.ft.com
+  aws.s3.bucket: com.ft.coco-imagepublish.pre-prod
```
Comment (tamas-molnar) [inline global-config/global-config.yaml]: Maybe rename this to be image-specific? `s3.image.binaries.bucket` or something similar?

File: kube-config-files/methode-image-model-mapper-service.yaml (hunk 3 above)
Comment (MMoisa) [reply kube-config-files/methode-image-model-mapper-service.yaml]: It's used by the binary-writer as well.

**Review by tamas-molnar — APPROVED** [review]

**Review by sbuliarca — APPROVED** [review]

**Review by davidbalazs93 — APPROVED** [review]

**Pull request merged**